docs/content/author-bundles.md

```diff
@@ -38,6 +38,10 @@ version: 0.1.0
 registry: getporter
 reference: getporter/azure-wordpress
 dockerfile: dockerfile.tmpl
+maintainers:
+- name: "John Doe"
+  email: "john.doe@mail.com"
```

Comment:
This is really minor, but can you switch this to use "example.com" for any fake email addresses and websites? That is a safer domain to use and won't accidentally ever be a real live email or website.

Reply:
Sure